### PR TITLE
rocksdb: Expose track-and-verify-wals-in-manifest config

### PR DESCRIPTION
**Pull request by v01dstar**
### What is changed and how it works?

Issue Number: Ref #16549

What's Changed:

```commit-message
Expose track-and-verify-wals-in-manifest config.

For further investigating corrupted WAL issue happened during EBS restore process.

```

### Related changes

- [ ] PR to update `pingcap/docs`/`pingcap/docs-cn`:
- [ ] Need to cherry-pick to the release branch

### Check List

Tests 

- [x] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

### Release note

```release-note
None
```

### CODE REVIEW

File: src/config/mod.rs [hunk 1]
```diff
@@ -1368,6 +1370,7 @@ impl Default for DbConfig {
             lockcf: LockCfConfig::default(),
             raftcf: RaftCfConfig::default(),
             titan: TitanDbConfig::default(),
+            track_and_verify_wals_in_manifest: false,
```
Comment (Connor1996) [inline src/config/mod.rs:1373]: it should be moved up

Comment (michaelmdeng) [inline src/config/mod.rs:1373]: more of a question, but given that [rocks recommends setting this in production](https://github.com/facebook/rocksdb/wiki/Track-WAL-in-MANIFEST), what are the implications of setting the default here to true?

Comment (v01dstar) [reply src/config/mod.rs:1373]: This is simply to keep the behavior same as before. But you are right, maybe it is better to make it default true. What's your opinion? @tonyxuqqi

Comment (tonyxuqqi) [reply src/config/mod.rs:1373]: @v01dstar In existing release branch's CP, we should make default value false. In master branch, I think it can be true.

Comment (v01dstar) [reply src/config/mod.rs]: Done

Comment (v01dstar) [reply src/config/mod.rs]: Done

**Review by Connor1996 — APPROVED** [review]
LGTM

**Pull request merged**